.github/workflows/release.yaml

```diff
@@ -0,0 +1,75 @@
+name: Create a release
+
+env:
+  BRANCH: release
+  VERSION: 3.3.1
+
+on:
+  schedule:
+    - cron: '30 10 * * 1'
+
+jobs:
+  build:
+    name: Check build condition for Snap
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version:
+          - '3.9'
+          - '3.10'
+    steps:
+      - id: checkout
+        name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.BRANCH }}
+      - id: setup-python
+        name: Setup Python
+        uses: actions/setup-python@v2.2.2
+        with:
+          python-version: ${{matrix.python-version}}
+          architecture: x64
+      - id: code-checks
+        name: Make sure the code follows conventions and tests correctly
+        run: |
+          make checks
+      - id: build-package
+        name: Build package
+        run: |
+          poetry build
```

Comment:
assuming this is different from "make build" that we do for the snap.

Reply:
yes, it is different. In the SNAP, with build, we build the snap, not the package

Reply:
:+1: